docs/glossary.md

```diff
@@ -0,0 +1,132 @@
+
+### Lighthouse 
+
+The [Lighthouse](https://github.com/decentraland/lighthouse) is the name used t reference the previous version of the Communications Service hosted on the Catalyst nodes and used to group users in a P2P network. 
+Main responsibilities were to determine which are the candidates for a P2P connection and do the WebRTC signaling to establish the connection. Most of this logic was done through two external components: the PeerJS Server (connects [WebRTC](https://webrtc.org/) peers) and Archipelago (receives users positions and groups them in islands).
+
+### Backend for Frontend (aka BFF) 
+
+The [BFF](https://github.com/decentraland/explorer-bff) Service was created to resolve client needs and to enable faster development of new features without breaking the existing APIs. In the context of the Catalyst Communication Service, it will be in charge of managing the P2P signaling and thus replacing the Lighthouse.
+
+### Catalyst Owner
+
+A [Catalyst Owner](https://github.com/decentraland/catalyst-owner) is an individual or organization running a Catalyst node. To add a Catalyst node to the DAO network a [proposal](https://governance.decentraland.org/?type=catalyst) needs to be submitted and the DAO nodes represent the trusted Decentraland Network.
+
+
+### Archipelago Service 
+
+The [Archipelago Service](https://github.com/decentraland/archipelago-service) previously was a library used by the Lighthouse. This service will have the same responsibility that the library did: group Peers in clusters so they can communicate efficiently and each cluster is called an island. On the other hand, the service also needs to be able to balance islands using the available communications Transports and following a set of [Catalyst Owner](https://github.com/decentraland/catalyst-owner) defined rules, in order to, for example, use LiveKit for an island in the Casino scene and P2P in a Genesis Plaza.
+
+### Island
+
+Cluster of connected peers.
+
+### Communication Transport 
+
+Transport is the technology used to share updates between Peers connected to an Island.
+
+### P2P Transport 
+
+Uses [WebRTC](https://webrtc.org/) connections between Peers, no extra infrastructure needed to scale as the load is on the client side and the Servers only manage the signaling. Latency increases as more Peers are added to an Island. There is a limit on the amount of WebRTC connections a client can have thus in order to have bigger islands, more indirect connections (jumps) between peers are needed.
+
+### LiveKit Transport 
+
+[LiveKit](https://livekit.io/) is an open source project that provides scalable, multi-user conferencing over WebRTC. Instead of doing a P2P network, Peers are connected to a Selective Forwarding Unit (SFU) in charge of managing message rely and different quality aspects of the communication. The SFUs are the added infrastructure needed in order to provide high-performance/high-quality communications between crowds on designated scenes.
```

Comment:
```suggestion
[LiveKit](https://livekit.io/) is an open source project that provides scalable, multi-user conferencing over WebRTC. Instead of doing a P2P network, Peers are connected to a Selective Forwarding Unit (SFU) in charge of managing message relay and different quality aspects of the communication. The SFUs are the added infrastructure needed in order to provide high-performance / high-quality communications between crowds on designated scenes.
```